examples/hosts/app-integration/external-data/src/externalData.ts

```diff
@@ -54,7 +63,7 @@ export interface IExternalDataSourceEvents extends IEvent {
  * on the server-side bot approach.  But using localStorage is probably good enough for the broadcast signal portion.
  */
 export class ExternalDataSource extends TypedEventEmitter<IExternalDataSourceEvents> {
-    public constructor() {
```

Comment:
TBH I'd like to see us reverse this lint rule - we use private constructors in a few places in the repo (boo) and I think it's nice to have it explicitly called out anyway.

Reply:
I tend to agree. Let me track down this rule and see if it is one we explicitly set or one we are inheriting.

Reply:
Hmm. This is a rule we explicitly set in the config. Probably merits further discussion outside of this context.

Reply:
Yeah I just disagree with it lol :-P

Reply:
But it looks like the results of that conversation never made it to the actual code -- the comment concludes we should do exactly what @ChumpChief suggested.

Reply:
I created a PR to update the strict config here: #13134